fcrepo/fcrepo
# Memento containment

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3450

# What does this Pull Request do?
Contains the past containment relationships for those things that did exist and are now deleted (but not purged).

**Note**: this does not include the repository rebuild processing. I figured I'd work that once this seemed correct.

**Note note**: I renamed the `resources` table to `containment` and the `transaction_operations` table to `containment_transactions`. The old names were not super clear and we have a lot more tables now.

# How should this be tested?

Create a resource (parent)
Create a child resource (child)
Create a version of parent (I patch it)
Delete child.

Get the memento you created, before there were no ldp:contains, now you should see child.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo/committers
